app/views/main.scala.html

```diff
@@ -12,6 +12,7 @@
     <link rel="stylesheet" media="screen" href='@routes.Assets.versioned("stylesheets/prism.css")'>
     <link rel="stylesheet" media="screen" href='@routes.Assets.versioned("stylesheets/Gov.css")'>
     <link rel="shortcut icon" type="image/png" href='@routes.Assets.versioned("images/favicon.png")'>
+    <script src="https://code.iconify.design/1/1.0.3/iconify.min.js"></script>
     <script src='@routes.Assets.versioned("javascripts/govuk-frontend-2.13.0.min.js")' type="text/javascript"></script>

```

Comment:
We're not keeping this library, it was just the quickest way of getting the icon